### PR TITLE
Christmas update - Snow planet station

### PR DESCRIPTION
**Pull request by PowerfulBacon**
## About The Pull Request

Adds in snow maps for christmas. Also makes it so the edge of the map doesn't need to be space turfs for transitions.
The snow map is more of a snow island because the borders need to be clean or the edge transition breaks.

image

Uses the exoplanet generation from supercruise.

Space station 13 is also replaced with winter outpost 13
image

## Why It's Good For The Game

It doesn't really feel very festive, given that its 2 days from christmas. This gets us all in the festive spirit by making our favourite space game wintery.

## Changelog
:cl:
add: The map will now be christmassy on christmas.
/:cl:

### CODE REVIEW

**Comment by Penwin0** [comment]
Maybe don't change the name to "Winter Planet" I can see that causing some confusion

File: code/modules/shuttle/super_cruise/orbital_map_components/orbital_objects/space_station.dm [hunk 1]
```diff
@@ -10,6 +10,9 @@
 /datum/orbital_object/z_linked/station/New()
 	. = ..()
 	SSorbits.station_instance = src
+	if(FESTIVE_SEASON in SSevents.holidays)
+		name = "Winter Planet"
```
Comment (PowerfulBacon) [inline code/modules/shuttle/super_cruise/orbital_map_components/orbital_objects/space_station.dm]:
```suggestion
		name = "Winter Outpost 13"
```

**Comment by AnCopper** [comment]
Bootleg IceBox.

Good work though, looks very nice.

**Comment by ghost** [comment]
Praise Crimbus.

**Comment by ike709** [comment]
Might need a little more polish for fullmerge, but I'm fine with leaving this testmerged for the next few days.

**Comment by PowerfulBacon** [comment]
This crashes with certain maps, untestmerging. This won't be worked on any more due to the time.

**Pull request closed without merging**